slither/printers/summary/printer_human_summary.py

```diff
@@ -0,0 +1,102 @@
+"""
+Module printing summary of the contract
+"""
+import logging
+
+from slither.detectors.detectors import Detectors
+from slither.printers.abstractPrinter import AbstractPrinter
+from slither.utils.code_complexity import compute_cyclomatic_complexity
+from slither.utils.colors import green, red, yellow
+
+
+class PrinterHumanSummary(AbstractPrinter):
+    ARGUMENT = 'print-human-summary'
+    HELP = 'Print an human readable summary of the contracts'
+
+    @staticmethod
+    def get_summary_erc20(contract):
+        txt = ''
+        functions_name = [f.name for f in contract.functions]
+        state_variables = [v.name for v in contract.state_variables]
+
+        if 'paused' in functions_name or 'pause' in functions_name:
```

Comment:
```suggestion
        if 'pause' in functions_name:
```

This check is enough as `pause` is in `paused` :)